### PR TITLE
Propagates dashboard port in standalone mode.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Propagates dashboard port in standalone mode.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #463 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by mukundansundar — COMMENTED** [review]
Add test for default port?

File: cmd/dashboard.go [hunk 1]
```diff
@@ -50,10 +50,9 @@ var DashboardCmd = &cobra.Command{
 			os.Exit(0)
 		}
 
-		if port < 0 {
-			localPort = defaultLocalPort
-		} else {
-			localPort = port
+		if dashboardLocalPort <= 0 {
+			print.FailureStatusEvent(os.Stdout, "Invalid port: %v", dashboardLocalPort)
```
Comment (vinayada1) [inline cmd/dashboard.go:54]: We need to document this change of behavior

**Comment by artursouza** [comment]
Default port comes from the cmd parsing, it is not UT.

File: cmd/dashboard.go (hunk 1 above)
Comment (artursouza) [reply cmd/dashboard.go:54]: Where do we document these things?

**Review by vinayada1 — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**